docs/content/Dialog2.mdx

```diff
@@ -0,0 +1,112 @@
+
+## Component props
+
+### DialogProps
+
+| Prop name     | Type                                        | Default            | Description                                                                                                   |
+| :------------ | :------------------------------------------ | :----------------- | :------------------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| title         | `string`                                    | `"Dialog"`         | Required. Sets the title of the dialog, which by default is also used as the `aria-labelledby` attribute.     |
+| subtitle      | `string`                                    | `string`           | Optional. Sets the subtitle of the dialog, which by default is also used as the `aria-describedby` attribute. |
+| renderHeader  | `(props: DialogHeaderProps) => JSX.Element` |                    | Optional. Custom render the dialog header. See "Custom rendering" above for more info.                        |
+| renderBody    | `(props: DialogProps) => JSX.Element`       |                    | Optional. Custom render the dialog body. See "Custom rendering" above for more info.                          |
+| renderFooter  | `(props: DialogProps) => JSX.Element`       |                    | Optional. Custom render the dialog footer. See "Custom rendering" above for more info.                        |
+| footerButtons | `DialogButtonProps[]`                       |                    | Optional. Specify buttons that will be rendered in the footer of the dialog.                                  |
+| onClose       | `(action: 'close-button'                    | 'escape') => void` |                                                                                                               | Required. This method is invoked when a gesture to close the dialog is used (either an Escape key press or clicking the "X" in the top-right corner). The gesture argument indicates the gesture that was used to close the dialog (either 'close-button' or 'escape'). |
+| role          | `"dialog"                                   | "alertdialog"`     | `"dialog"`                                                                                                    | The ARIA role given to the dialog element. More info: [dialog](https://www.w3.org/TR/wai-aria-practices-1.1/#dialog_modal), [alertdialog](https://www.w3.org/TR/wai-aria-practices-1.1/#alertdialog)                                                                    |
```

Comment:
Looks like you need to escape the `|` in the type column for both of these lines.  Got formatted as a column break.

Reply:
I changed all the `|` to U+2502 (`BOX DRAWINGS LIGHT VERTICAL`) since I literally could not figure out how to escape `|` inside a `code block` inside a table.